pages/_app.js

```diff
@@ -49,11 +51,16 @@ function Site({ Component, pageProps, authentication }) {
         type: pageProps.pageContext?.type || "",
       },
       breadcrumbs: pageProps.breadcrumbs || [],
+      linkReload: {
+        isEnabled: isLinkReloadEnabled,
+        setIsEnabled: setIsLinkReloadEnabled,
```

Comment:
This is a small thing, so I don't think should make or break the PR, but I wonder if we can make a name that's a little more clear what it's doing. Maybe something like `setReloadable`, or could we have a `disableReload()` and `enableReload()`.

Reply:
I think I’ll keep this as is. To me it’s important that _links_ cause a reload, where `setReloadable` seems _less_ specific about what this relates to. This way, you always see something like `linkReload.isEnabled` which reads just about like the English for what this means.